lib/internal/buffer.js

```diff
@@ -777,46 +785,63 @@ function writeFloatBackwards(val, offset = 0) {
   return offset;
 }
 
-// FastBuffer wil be inserted here by lib/buffer.js
+class FastBuffer extends Uint8Array {}
+
+function addBufferPrototypeMethods(proto) {
+  proto.readUIntLE = readUIntLE;
```

Comment:
Another way to do this is to put all these to the prototype of `FastBuffer`, I am not sure if there are any concerns about edge cases though

Reply:
I am fine either way.